packages/grid/_modules_/grid/hooks/features/rows/useGridEditRows.ts

```diff
@@ -302,25 +304,23 @@ export function useGridEditRows(
   useGridApiEventHandler(apiRef, GRID_CELL_DOUBLE_CLICK, handleCellDoubleClick);
   useGridApiEventHandler(apiRef, GRID_CELL_FOCUS_OUT, handleCellFocusOut);
   useGridApiEventHandler(apiRef, GRID_COLUMN_HEADER_DRAG_START, handleColumnHeaderDragStart);
-  useGridApiEventHandler(apiRef, GRID_CELL_EDIT_ENTER, handleCellEditEnter);
-  useGridApiEventHandler(apiRef, GRID_CELL_EDIT_EXIT, handleCellEditExit);
+  useGridApiEventHandler(apiRef, GRID_CELL_EDIT_START, handleCellEditEnter);
+  useGridApiEventHandler(apiRef, GRID_CELL_EDIT_STOP, handleCellEditExit);
   useGridApiEventHandler(apiRef, GRID_CELL_EDIT_COMMIT, handleCellEditCommit);
   useGridApiEventHandler(apiRef, GRID_EDIT_CELL_PROPS_CHANGE, handleEditCellPropsChange);
 
   useGridApiOptionHandler(apiRef, GRID_CELL_EDIT_COMMIT, options.onCellEditCommit);
-  // TODO remove, use onEditRowsModelChange directly
```

Comment:
Did we find cases where using `onEditCellPropsChange` is relevant over `onEditRowsModelChange` or can we still remove the `onEditCellPropsChange` prop?  I recall there was something about extra props that I didn't understand the use case.

Reply:
To do server-side validation `onEditCellPropsChange` is relevant. While we don't provide a reason argument we need to support this prop. In the mean time, I kept its usage low leveraging `onEditRowsModelChange` whenever I could.